test/benchmark_test.cc

```diff
@@ -82,16 +82,18 @@ BENCHMARK(BM_CalculatePi)->ThreadRange(1, 32);
 BENCHMARK(BM_CalculatePi)->ThreadPerCpu();
 
 static void BM_SetInsert(benchmark::State& state) {
+  std::set<int> data;
+  std::set<int> orig = ConstructRandomSet(state.range(0));
```

Comment:
this changes things a bit, as it means the data will be the same for every iteration. this may drop variance within a run but increase it across runs.

Reply:
oh. `ConstructRandomSet` is a lie. never mind then.